src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String name, String color) {
+        super(name, color);
+        this.radius = getRandomSide();
+    }
+
+    public Circle(String name, String color, double radius) {
+        super(name, color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return (Math.PI * Math.pow(radius, 2));
+    }
+
+    @Override
+    public void getDrawing() {
+        System.out.println("Figure: " + getName() + " , "
+                + "area: " + String.format("%.2f", getArea()) + " sq.units, "
+                + "radius: " + radius + " units, "
+                + "color: " + getColor() + ".");
+    }
+
+    @Override
+    public int getRandomSide() {

```

Comment:
you should generate random color and random values for your figures in FigureSupplier class, not in figures, fix it everywhere